src/diffusers/loaders.py

```diff
@@ -475,22 +407,47 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
                         cross_attention_dim=cross_attention_dim,
                     )
                     attn_processors[key].load_state_dict(value_dict)
+
+            self.set_attn_processor(attn_processors)
```

Comment:
This one corresponds to custom diffusion and seems like an irrelevant change.

Reply:
Yeah but we need to do this for custom diffusion

Reply:
It is not irrelevant in the sense that we now bring it only into the `is_custom_diffusion` clause